### ISSUE TITLE
Feature: A simple way for labeler to update the endpoint registered with PLC

### ISSUE THREAD

**Issue by jetpacktuxedo** [bluesky-social/ozone#104]
I recently migrated my labeler from one domain to another, and to update the URL on the PLC end I needed to manually construct, sign, and submit a PLC operation. Ideally there would something in the ozone configuration menu or something that I could click to tell my instance to push its new URL up to the repo.

In the meantime, [here is the kind of janky python script I used to do my update](https://gist.github.com/jetpacktuxedo/67a2fbed24633c3190fe051108e44e37)

**Comment by bnewbold** [comment]
Yeah, this has definitely been a pain point for folks setting up labelers.
